__tests__/rpcProvider.test.ts

```diff
@@ -128,9 +128,5 @@ describeIfRpc('RPCProvider', () => {
       expect(contractClass).toHaveProperty('program');
       expect(contractClass).toHaveProperty('entry_points_by_type');
     });
-
-    test.todo('getEstimateFee');
```

Comment:
One more thing is just to add on these methods a comment to not use them outside of the `Account` cc: @tabaktoni

Reply:
This should be added into src/provider/interface.ts and not RPC provider.
I think you can add to @deprecated note also this one:
`@deprecated Please use getInvokeEstimateFee or getDeclareEstimateFee instead. Should not be used outside of Account class`

Reply:
Ah ok, I just updated it